### PR TITLE
Update pktmon.md

### PR DESCRIPTION
**Pull request by george-guirguis**
-removed the mention of release codenames
-added the best practices section
-edited the limitations section

### CODE REVIEW

**Comment by PRMerger12** [comment]
@george-guirguis : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: WindowsServerDocs/networking/technologies/pktmon/pktmon.md [hunk 1]
```diff
@@ -39,21 +39,30 @@ Packet Monitor intercepts packets at multiple locations throughout the networkin
...
 </center>
 
+## Best Practices
+
+Use these best practices for streamlining your network analysis.
+
+-Check command-line help for arguments and capabilities (e.g. pktmon start help).
+-Configure packet filters matching your scenario (pktmon filter add).
+-Check packet counters during the experiment for high level view (pktmon counters).
+-Look into the log for detailed analysis (pktmon format pktmon.etl).
```
Comment (illfated) [inline WindowsServerDocs/networking/technologies/pktmon/pktmon.md:49]: Looks like you intended to use a bullet point list here. Let's make that list work as intended by adding the required spacing between the list indicators and their following text:
```suggestion
- Check command-line help for arguments and capabilities (e.g. pktmon start help).
- Configure packet filters matching your scenario (pktmon filter add).
- Check packet counters during the experiment for high level view (pktmon counters).
- Look into the log for detailed analysis (pktmon format pktmon.etl).
```

Comment (illfated) [reply WindowsServerDocs/networking/technologies/pktmon/pktmon.md:49]: By the way, you can test the bullet point list format by adding the same single spaces in your main comment above:

> - removed the mention of release codenames
> - added the best practices section
> - edited the limitations section

**Review by khdownie — APPROVED** [review]

**Comment by illfated** [comment]
@george-guirguis : Would you be able to add my suggested change, to make that bullet point list work as intended?

**Review by george-guirguis — COMMENTED** [review]
.

**Comment by PRMerger9** [comment]
@george-guirguis : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: WindowsServerDocs/networking/technologies/pktmon/pktmon.md [hunk 2]
```diff
@@ -62,9 +71,8 @@ Packet Monitor's functionality has evolved through Windows releases. The table b
 
 Below are some of Packet Monitor's most significant limitations.
 
-- Wireless media support is available on Iron builds only
-- No firewall integration
-- Drop reporting is only available for integrated components
+- Ethernet traffic only is supported for now. Wireless traffic support will be added in later releases.
```
Comment (illfated) [inline WindowsServerDocs/networking/technologies/pktmon/pktmon.md:74]: Minor grammatical improvement suggestion:
```suggestion
- Only Ethernet traffic is supported for now. Wireless traffic support will be added in later releases.
```
Alternate sentence structure (but maybe causing a different interpretation):
> Ethernet traffic is the only supported option for now. Wireless traffic support will be added in later releases.

**Review by khdownie — APPROVED** [review]

**Comment by PRMerger18** [comment]
@george-guirguis : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by khdownie — APPROVED** [review]

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by illfated** [comment]
#sign-off

**Comment by PRMerger17** [comment]
@illfated: I'm sorry - only the author of this article, @khdownie, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by illfated** [comment]
Yes, I know. I just added the comment (as agreed) to get more attention to this PR.

**Comment by khdownie** [comment]
#sign-off

**Review by eross-msft — APPROVED** [review]

**Pull request merged**